script.skin.info.service/Utils.py

```diff
@@ -143,24 +128,29 @@ def set_artist_properties(audio):
     firstyear = 0
     playcount = 0
     for item in audio['result']['albums']:
+        art = item['art']
         HOME.setProperty('SkinInfo.Artist.Album.%d.Title' % count, item['title'])
         HOME.setProperty('SkinInfo.Artist.Album.%d.Year' % count, str(item['year']))
-        HOME.setProperty('SkinInfo.Artist.Album.%d.Thumb' % count, item['thumbnail'])
         HOME.setProperty('SkinInfo.Artist.Album.%d.DBID' % count, str(item.get('albumid')))
         HOME.setProperty('SkinInfo.Artist.Album.%d.Label' % count, item['albumlabel'])
+        HOME.setProperty('SkinInfo.Artist.Album.%d.Art(discart)' % count, art.get('discart', ''))
+        HOME.setProperty('SkinInfo.Artist.Album.%d.Art(thumb)' % count, art.get('thumb', ''))
         if item['playcount']:
             playcount = playcount + item['playcount']
         if item['year'] > latestyear:
             latestyear = item['year']
         if firstyear == 0 or item['year'] < firstyear:
             firstyear = item['year']
         count += 1
-    if firstyear > 0 and latestyear < 2020:
+    if firstyear > 0 and latestyear < 2030:
         HOME.setProperty('SkinInfo.Artist.Albums.Newest', str(latestyear))
         HOME.setProperty('SkinInfo.Artist.Albums.Oldest', str(firstyear))
     HOME.setProperty('SkinInfo.Artist.Albums.Count', str(audio['result']['limits']['total']))
     HOME.setProperty('SkinInfo.Artist.Albums.Playcount', str(playcount))
 
+    if ADDON.getSetting("enable_debug_json") == "true":
```

Comment:
use ` xbmcaddon.Addon([id]).getSettingBool(id)`, https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html#gaf0205dd5b96354f30a39fbf35599e681

Reply:
@Rechi Shouldn't changing it to the below code work?
```if ADDON.getSettingBool("enable_debug_json") == "true":```


It doesn't work for me, I must be missing something obvious here. 
Any pointers would be most appreciated.

Thanks 😃

Reply:
`if ADDON.getSettingBool("enable_debug_json"):`

Reply:
Of course... Should have tried that. Thanks!
I will update it and cleanup the commented out code and do a force push in the next few min.